src/dmd/hdrgen.d

```diff
@@ -3962,3 +3962,152 @@ private void typeToBufferx(Type t, OutBuffer* buf, HdrGenState* hgs)
         case Ttag:       return visitTag(cast(TypeTag)t);
     }
 }
+
+/****************************************
+ * Convert EXP to char*.
+ */
+
+string EXPtoString(EXP op)
+{
+    __gshared immutable char*[EXP.max + 1] strings =
```

Comment:
dmd/src/dmd/hdrgen.d(3972:44)[warn]: Variable declaration for `strings` has redundant attributes (`__gshared`, `immutable`)

Reply:
@WalterBright I see that you made another push, however, the buildkite failure persists as this was not fixed.

Reply:
```suggestion
    static immutable char*[EXP.max + 1] strings =
```